google/cloud/aiplatform/prediction/local_model.py

```diff
@@ -251,6 +252,7 @@ def build_cpr_model(
                 handler, src_dir
             )
         environment_variables = {
+            "CPR_USER_DIR_NAME": Path(src_dir).name,
```

Comment:
Why do we need to do Path().name here?

Reply:
If users provide "tmp/src_dir", we check all the necessary code (predictor, handler, and requirements.txt) should be in "tmp/src_dir" and copy "src_dir" to the workdir in the images.

Reply:
Ah, understood, thanks.